monkey/monkey_island/cc/services/config.py

```diff
@@ -360,7 +360,7 @@ def _encrypt_or_decrypt_config(config, is_decrypt=False):
                 parent_config_arr = config_arr
                 config_arr = config_arr[config_key_part]
 
-            if isinstance(config_arr, collections.Sequence) and not isinstance(config_arr, str):
+            if isinstance(config_arr, collections.abc.Sequence) and not isinstance(config_arr, str):
```

Comment:
What's the rationale behind this?

Reply:
![image](https://user-images.githubusercontent.com/19957806/126310120-14f6d6a8-a480-4473-883b-3cfff98cddab.png)